cli/jobs/pipelines/iris-add-column-and-word-count-using-spark/pipeline.yml

```diff
@@ -0,0 +1,50 @@
+type: pipeline
+
+description: 'submit a spark job using component sdk'
+
+inputs:
+  iris_data:
+    type: uri_file
+    path: ./dataset/iris.csv
+    mode: direct
+
+outputs:
+  output:
+    type: uri_folder
+    mode: direct
+
+jobs:
+  spark_job:
```

Comment:
I think we should have a sample with node that at least train a model

Reply:
add spark kmeans node